metadata-ingestion/src/datahub/ingestion/reporting/datahub_ingestion_run_summary_provider.py

```diff
@@ -146,12 +146,53 @@ def __init__(self, sink: Sink, report_recipe: bool, ctx: PipelineContext) -> Non
             aspect_value=source_info_aspect,
         )
 
+    @staticmethod
+    def convert_sets_to_lists(obj: Any) -> Any:
```

Comment:
```suggestion
    def _convert_sets_to_lists(obj: Any) -> Any:
```

Reply:
If I make this private then how will I be able to test this? I know it is accessible but seems wrong to test internal methods? Or is it to avoid making it part of the interface?

Reply:
yup this is to avoid making it part of the interface

also ideally we'd call this method `_prepare_for_json_dump()` or something to make it more clear what the goal is